pkg/file/copy.go

```diff
@@ -0,0 +1,55 @@
+package file
```

Comment:
I wouldn't have a package for this- I'd put this as a function inside the `cmd/theatre-envconsul/main.go` personally. It won't be used anywhere other than that!

On an aside, christ this is totally horrible. Why is this not simpler.

Reply:
I guess everyone has different needs from a copy command? And then handling all the edge cases is just verbose. But like you mention below, we know what files we are copying, so we can get away with assuming things.